frontend/packages/console-app/src/components/nodes/NodesPage.tsx

```diff
@@ -402,9 +401,12 @@ const NodesTable: React.FC<NodesTableProps> = React.memo((props) => {
     [],
   );
   const { t } = useTranslation();
+  const [tableColumns, , loaded] = useTableColumns(columnManagementID);
```

Comment:
I'm not sure, but do we need a blank `updateTableColumn` or can we return/alter it in the `useTableColumns` definition return statement